package.json

```diff
@@ -17,6 +17,7 @@
     "pretest": "npm run build",
     "test": "vitest",
     "build-types": "openapi-typescript https://staging-api-atlas.nomic.ai/v1/api-reference/openapi.json -o ./src/type-gen/openapi.ts --empty-objects-unknown | npx prettier ./src/type-gen/openapi.ts --write",
+    "build-types-local": "openapi-typescript http://localhost:8080/v1/api-reference/openapi.json -o ./src/type-gen/openapi.ts --empty-objects-unknown | npx prettier ./src/type-gen/openapi.ts --write",
```

Comment:
my only concern with this is it makes it easier to put up types that dont actually comport with our openapi on staging, and it would be impossible for a reviewer to know